kedro/framework/cli/catalog.py

```diff
@@ -179,6 +179,18 @@ def create_catalog(metadata: ProjectMetadata, pipeline_name, env):
         click.echo("All datasets are already configured.")
 
 
+@catalog.command("factories")
+@env_option
+@click.pass_obj
+def list_patterns(metadata: ProjectMetadata, env):
+    "Show all factory patterns in catalog, ranked by priority by which they are matched"
```

Comment:
Nit:
```suggestion
    "Show all factory patterns in the catalog, ranked by priority by which they are matched."
```

Reply:
Changed in [4aa5bf8](https://github.com/kedro-org/kedro/pull/2796/commits/4aa5bf870d1737e1e9cd18bdfd6a90ef30913742)